cpp/locality/NeighborList.cc

```diff
@@ -77,12 +82,11 @@ void NeighborList::updateSegmentCounts() const
     {
         m_counts.prepare(m_num_query_points);
         m_segments.prepare(m_num_query_points);
-        int index(-1);
         int last_index(-1);
         unsigned int counter(0);
         for (unsigned int i = 0; i < getNumBonds(); i++)
         {
-            index = m_neighbors(i, 0);
+            int index = m_neighbors(i, 0);
```

Comment:
This is `const` within the body of the loop.
```suggestion
            const int index = m_neighbors(i, 0);
```

Reply:
@vyasr This is marked as resolved but I don't see the change. Did I make a mistake here?

Reply:
Just not pushed yet. I'll push shortly.